.github/workflows/pypi.yml

```diff
@@ -1,8 +1,7 @@
-name: Build and publish to PyPI
+name: "[Deprecated] Build and publish to PyPI"
 
-on:
-  release:
-    types: [published]
+# runnable manually
+on: workflow_dispatch
```

Comment:
If left to automatically triggering on published releases, this would probably interfere with the new CD pipeline.
So for now switched to manual triggering, to still allow manual releases.
If from now on manual releases are deemed unnecessary, this file should be dropped altogether.